site/source/docs/compiling/WebAssembly.rst

```diff
@@ -56,6 +65,14 @@ There are some differences you may notice between the two backends, if you upgra
     LLVM optimization passes by default, even if using LLVM IR in object files;
     for that you must pass ``--llvm-lto 1``.
 
+* `wasm-ld`, the linker used by the wasm backend, requires librarys (`.a`
+  archives) to continue symbol indexes.  This matches the behaviour the native
+  GNU linker.  While `emar` will create such indexes by default, native tools
+  such as GNU `ar` and GNU `strip` are not aware of the WebAssembly object
+  format and cannot create archive indexes.  In particular if you run GNU
+  `strip` on an archive file contains WebAssembly abject files it will remove
```

Comment:
"archive file that contains", and abject->object

Reply:
Right.. i wouldn't want to pass judgement on this miserable abject files!